templates/named.conf.erb

```diff
@@ -24,5 +24,12 @@ acl "<%= acl_name %>"  {
 };
 <%- end -%>
 
+<%- if scope.lookupvar('::dns::additional_directives').any? -%>
+// additional directives
+<% scope.lookupvar('::dns::additional_directives').each do |directive| -%>
+<%= directive %>
+<%- end -%>
+<%- end -%>
```

Comment:
You are right, I updated the PR source after some apply tests. Now, without additional_directives, the named.conf stays as it was.